lib/random.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * Picks a random item from an array.
+ *
+ * @template {T}
+ * @param {T[]} arr
+ * @returns {T}
+ */
+export const pickRandomItem = (arr) => {
+  return arr[Math.floor(Math.random() * arr.length)]
+}
```

Comment:
This function should rather be inlined in where we need it, and that is only inside `main.js`. To me, a single randomness function doesn't justify creating a new lib file

Reply:
Done in https://github.com/CheckerNetwork/walrus-checker/pull/12/commits/f880b55feea819e835df1868583d49f9b5585cc2. Thanks for feedback!